src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx

```diff
@@ -241,17 +247,18 @@ To avoid these issues, set wait conditions that will pause the script until the
   >
     Instruct the monitor to wait for a page element to be present. This is useful when an element your script depends on takes a long time to load, or simply to ensure that a page element is present before your script clicks on it.
 
-    Use any of the [locate functions](#locating-elements) with the wait function below. Use your desired function instead of the `$driver.By.linkText` function used below:
+    Use any of the [locate functions](#locating-elements) with the wait function below. Use your desired function instead of the `$selenium.By.linkText` function used below:
 
     ```
     //Tell the monitor to wait up to 7,500 milliseconds (7.5 seconds) for a link whose text is `Your Link Text Here`.
-    $browser.waitForAndFindElement($driver.By.linkText("Your Link Text Here"), 7500);
+    var element = await $webDriver.wait($selenium.until.elementLocated($selenium.By.linkText("Your Link Text Here")), 7500, 'Timed-out waiting for element to be located using: '+locator);
+    await $webDriver.wait($selenium.until.elementIsVisible(element), 7500, 'Timed-out waiting for element to be visible using ${element}');
```

Comment:
In the netlify preview this section looks pretty cramped and awful (not even including the error in the script I left in). 

I'm going to prettify this, but it will end up looking a bit different to the other scripts in the page - but way more readable so that's probably okay.